network-api/networkapi/wagtailpages/migrations/0004_auto_20200714_2223.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.13 on 2020-07-14 22:23
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailimages', '0022_uploadedimage'),
```

Comment:
`('wagtailimages', '0022_uploadedimage'),` caused an `django.db.migrations.exceptions.NodeNotFoundError: Migration wagtailpages.0004_auto_20200714_2223 dependencies reference nonexistent parent node ('wagtailimages', '0022_uploadedimage')` error. 

Commented it out and it works. May need to regenerate this migration or remove line 11.

Reply:
is this still the case after the whole docker prune / new-env?

Reply:
No it's not. Ill resolve this.